extensions/ql-vscode/package.json

```
@@ -1969,6 +1969,7 @@
   },
   "dependencies": {
     "@floating-ui/react": "^0.26.10",
+    "@octokit/plugin-throttling": "^8.0.0",
     "@octokit/plugin-retry": "^6.0.1",
```

I believe these two lines should be swapped to follow alphabetical order (like in the `package-lock.json`).

```suggestion
    "@octokit/plugin-retry": "^6.0.1",
    "@octokit/plugin-throttling": "^8.0.0",
```

---

I can't do alphabet 😂